lib/httpcheck.js

```diff
@@ -70,6 +70,7 @@ var HttpChecker = {
 			if ( pointerCurrentMax > arrCheck.length )
 				pointerCurrentMax = arrCheck.length;
 			for ( ; pointer < pointerCurrentMax ; pointer++ ) {
+				logger.debug('checkServers', arrCheck[ pointer ]);
```

Comment:
Do we need this here? If yes, maybe wrap it in a check whether DEBUG is on?

Reply:
We don't, nice catch :)